### PR TITLE
Add private guest lobby messages

### PR DESCRIPTION
**Pull request by SashoVihVas**
### What does this PR do?
This PR takes what has been implemented in the first place in #11593 by @pedrobmarin  and extends it with the possibility of sending personalized guest lobby messages. After every item concerning the information about a guest user, a personal text input field could be made visible by clicking on the new label "Message" right next to the user's name. A personal text message could be sent to the waiting user. Then, if the text input field bothers the moderator, it could be reverted by clicking again on "Message" so that the text input field disappears. Instead of the guest lobby message for all, the targeted waiting user will see just the message sent to him by the moderator. This could be reverted by sending just an empty string in the personal text input field and the targeted user will once again see the guest lobby message that displays for all. By default, if there is no text input for a targeted user, all of the waiting users see the common guest lobby message.

private_messages

### Motivation
As a tutor at the university I also proctor exams via BBB. Admissions in the conference room happen, when specific requirements are met. For example: In an online-supervised exam via BBB a student should join with his full name instead of matriculation number or something else. It would be nice if the moderator could specifically write to this student beforehand, while in the guest lobby to warn him.

### CODE REVIEW

File: akka-bbb-apps/src/main/scala/org/bigbluebutton/core/models/GuestsWaiting.scala [hunk 1]
```diff
@@ -56,6 +62,15 @@ class GuestsWaiting {
   def setGuestPolicy(policy: GuestPolicy) = guestPolicy = policy
 
   def setGuestLobbyMessage(message: String) = guestLobbyMessage = message
+
+  def setPrivateGuestLobbyMessage(intId: String, message: String): Unit = {
+    if (guestsWithPrivateGuestLobbyMessages.contains(intId)) {
+      guestsWithPrivateGuestLobbyMessages.remove(intId);
+      guestsWithPrivateGuestLobbyMessages.put(intId, message);
+    } else {
+      guestsWithPrivateGuestLobbyMessages.put(intId, message);
```
Comment (pedrobmarin) [inline akka-bbb-apps/src/main/scala/org/bigbluebutton/core/models/GuestsWaiting.scala:71]: Check if this is really needed for mutable HashMaps

File: bbb-common-web/src/main/java/org/bigbluebutton/api/domain/Meeting.java [hunk 2]
```diff
@@ -375,10 +377,21 @@ public void setGuestLobbyMessage(String message) {
...
+			return usersWithGuestLobbyMessages.get(guestId);
+		}
 		return guestLobbyMessage;
 	}
 
+	public void setPrivateGuestLobbyMessage(String guestId, String message) {
+		if (usersWithGuestLobbyMessages.containsKey(guestId)) {
+			usersWithGuestLobbyMessages.replace(guestId, message);
+		} else {
+			usersWithGuestLobbyMessages.put(guestId, message);
```
Comment (pedrobmarin) [inline bbb-common-web/src/main/java/org/bigbluebutton/api/domain/Meeting.java:391]: Same `Map` question.

**Review by pedrobmarin — COMMENTED** [review]
Looks good! Some minor things that might be improved at the code. I'll run and make extra tests over it in the next few days.

**Comment by SashoVihVas** [comment]
Hi @pedrobmarin, HashMap updates are corrected as suggested!

**Review by pedrobmarin — APPROVED** [review]

**Comment by antobinary** [comment]
Well done, @SashoVihVas :partying_face: Thank you for all these Guest Lobby improvements!
Thank you @pedrobmarin for the thorough reviews!

**Pull request merged**